Formula/pyside.rb

```diff
@@ -48,17 +59,22 @@ def install
 
   test do
     system Formula["python@3.9"].opt_bin/"python3", "-c", "import PySide2"
-    %w[
+    modules = %w[
       Core
       Gui
       Location
       Multimedia
       Network
       Quick
       Svg
-      WebEngineWidgets
       Widgets
       Xml
-    ].each { |mod| system Formula["python@3.9"].opt_bin/"python3", "-c", "import PySide2.Qt#{mod}" }
+    ]
+
+    # QT web engine is currently not supported on Apple
+    # silicon. Re-enable it once it has been enabled in the qt.rb.
+    modules << "WebEngineWidgets" unless Hardware::CPU.arch == :arm64
```

Comment:
```suggestion
    modules << "WebEngineWidgets" unless Hardware::CPU.arm?
```
Sorry, just caught this...